apps/fabric-website/index.html

```diff
@@ -1,157 +1,203 @@
+              e = window;
+            setTimeout(function() {
+              var b = d.createElement('script');
+              (b.src = a.url || 'https://az416426.vo.msecnd.net/scripts/a/ai.0.js'),
+                d.getElementsByTagName('script')[0].parentNode.appendChild(b);
+            });
+            try {
+              c.cookie = d.cookie;
+            } catch (a) {}
+            c.queue = [];
+            for (var f = ['Event', 'Exception', 'Metric', 'PageView', 'Trace', 'Dependency']; f.length; ) b('track' + f.pop());
+            if (
+              (b('setAuthenticatedUserContext'),
+              b('clearAuthenticatedUserContext'),
+              b('startTrackEvent'),
+              b('stopTrackEvent'),
+              b('startTrackPage'),
+              b('stopTrackPage'),
+              b('flush'),
+              !a.disableExceptionTracking)
+            ) {
+              (f = 'onerror'), b('_' + f);
+              var g = e[f];
+              e[f] = function(a, b, d, e, h) {
+                var i = g && g(a, b, d, e, h);
+                return !0 !== i && c['_' + f](a, b, d, e, h), i;
+              };
+            }
+            return c;
+          })({
+            instrumentationKey: '18e07455-6fee-47f7-ab59-8dbf99a31d23'
+          });
+
+        // @TODO: Add custom referrer tracking here. https://stackoverflow.com/questions/5097808/is-document-referrer-cross-browser-compatible
+        (window.appInsights = appInsights), appInsights.queue && 0 === appInsights.queue.length && appInsights.trackPageView();
+      </script>
+    -->
+  </head>
+
+  <body>
+    <div id="main">
+      <div class="loading">
+        <img
+          class="loadingImage"
+          src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/spinner.gif"
+          alt="Loading"
+          width="32"
+          height="32"
+        />
+      </div>
     </div>
-  </div>
-
-  <script type="text/javascript">
-    var isLocal = window.location.hostname === 'localhost' || window.location.hostname.indexOf('ngrok.io') > -1;
-    var isDogfood = false;
-    var isProduction = false;
-
-    // Query params
-    var dogfoodParam = getParameterByName('isDogfood');
-    var productionParam = getParameterByName('isProduction');
-    var noCompress = isLocal || getParameterByName('noCompress');
-    var devhost = getParameterByName('devhost');
-
-    var entryPointFilename = 'fabric-sitev5';
-    var appPath = '';
-    var Flight = {};
-
-    // Determine production or staging/dogfood
-    if (location.hostname === 'developer.microsoft.com' || productionParam) {
-      isProduction = true;
-    } else if (location.hostname === 'uifabric-tst.azurewebsites.net' || dogfoodParam) {
-      isDogfood = true;
-    }
-
-    // If the site is hosted, load the flight config script for production or dogfood
-    if (!isLocal) {
-      // Update as new fabric versions and its documentation are supported
-      var fabricVersions = [5, 6];
```

Comment:
The fabricVersion stuff should stick around--it's used to enable version switching.